userspace/falco/app/actions/load_config.cpp

```diff
@@ -66,6 +66,10 @@ falco::app::run_result falco::app::actions::load_config(const falco::app::state&
 		}
 	}
 
+	s.config->m_falco_reload_ts = (int64_t)std::chrono::duration_cast<std::chrono::nanoseconds>(
```

Comment:
What if we set this value in the configuration object constructor? After all it is bound to the lifetime of the config object, right?
It should work fine for our use case i think.

Reply:
Since it's not user configurable I would leave it w/ the comment for now and perhaps we can re-evaluate the entire metrics approach for the next release? In Falco it is still a bit rough around the edges, I believe in libs we have cleaned it up a lot and it's in a better state there.